src/Runtime/Runtime/System.Windows.Controls/DatePicker.cs

```diff
@@ -404,6 +404,7 @@ private void ProcessTextBox()
                     }
                     else
                     {
+                        _textBox.Text = string.Empty;
```

Comment:
Should we update the Text property instead ? (it will update _textBox in the dependency property callback)
```suggestion
                        SetValue(TextProperty, string.Empty);
```

Reply:
i tried it, it doesn't clear the textbox